### DISCUSSION TITLE
Only the first unread article of a series is marked as unread

### DISCUSSION THREAD

**Discussion by PiperKev** [martinrotter/rssguard#1346] [Q&A] [answered]
I don't know if I accidentally changed something or not, but the behavior of RSS Guard is different than when I first installed it.  Initially, all unread articles in my feed were displayed as unread, but after I installed 4.6.6, only the first unread article is so marked.  The rest are already displayed as read.  Is there a setting that will revert the behavior back to what it was?
![Screenshot from 2024-03-21 08-53-58](https://github.com/martinrotter/rssguard/assets/8465037/8ceb3fac-d69b-477a-ac8b-82d018106fc8)

**Comment by martinrotter** [comment]
Not sure what exactly you mean. I see bunch of read articles + 1 unread article which is what I would normally expect. If you want to only see unread articles, then there is article list filter button above article list which allows to filter out unwanted articles.

**Reply by PiperKev** [reply]
Right, I guess I should clarify.  The feed I subscribe to is an Atom feed for a group that produces ebooks and, when they update existing ones or add new ones, their feed creates new articles.  In the past, say for example they updated 5 books, the 5 articles would be displayed as unread.  Now, only the first of the 5 display as unread, and the subsequent 4 are marked as read.  Am I making sense?  It's not a huge thing, and I LOVE RSS Guard, but I was wondering if maybe I had inadvertently changed a setting?

**Reply by PiperKev** [reply]
THIS is how it used to look when I fetched articles and there were new books released:

![Screenshot from 2024-03-21 09-29-10](https://github.com/martinrotter/rssguard/assets/8465037/5284897e-5691-441e-bd65-9dfeae3122fc)

**Reply by martinrotter** [reply, marked as answer]
Let me explain.

If there is a NEW article/item in the feed, it starts in RSS Guard as UNREAD (as everyone would expect).

If some article is UPDATED, then RSS Guard starting with version 4.6.4 DOES KEEP existing read/unread state.

Older RSS Guard version <4.6.4 were automatically marking UPDATED articles as UNREAD which some users found weird.

Now, once you mark some article as read, it stays read even if its contents is updated.

------

That could explaing the behavior you are seeing, maybe? It does not make sense to me why 1 article was marked unread tho.

**Reply by PiperKev** [reply]
Ah, that makes perfect sense!  Thank you!  The reason that 1 article was marked unread was because it was a new book.  The rest were updates to existing books, so those articles being marked as read makes sense given your explanation!  i can't remember which version I originally installed, but it must've been <4.6.4 .  Thanks again!
